distro/README.md

```diff
@@ -11,6 +11,20 @@ function to create an SDK that is ready to be used with OpenTelemetry and
 forward all telemetry to Splunk. See [`example_test.go`](./example_test.go) for
 a complete example.
 
+## Logging
+
+By default the [`SDK`] will log relevant operation information, warnings, and
+errors. This logging can be explicitly configured in the code that creates the
+[`SDK`] using the `WithLogger` option to configure a logger that only logs what
+is wanted. Or, the default logging level can be set using the OpenTelemetry
+defined `OTEL_LOG_LEVEL` environment variable. The following values for that
+variable are understood.
+
+- `"error"`: only log errors
+- `"warn"`: log errors and warnings
+- `"info"`: log operation information, warnings, and errors
+- `"debug"`: log debugging and operation information, warnings, and errors
+
 ## Configuration
 
 The [`SDK`] is configured with the following options.
```

Comment:
We usually don't mention the SDK in other distros's repos. I assume this just describes our distribution defaults?

Reply:
Yes, this specifically describes option types that only our distro accepts and the unique defaults we have.